popSimManu.tex

```diff
@@ -667,14 +704,20 @@ \subsection*{Model QC procedures}
 into \stdpopsim that allows for rigorous quality control. Imagine Developer A
 wants to introduce a new model into \stdpopsim. Developer A implements the
 demographic model for the relevant organism along with clear documentation
-of the model parameters and populations. This model is submitted as a Pull Request,
+of the model parameters and populations. 
+% ALG: I’m not sure what documentation of populations means. 
+This model is submitted on GitHub as a Pull Request,
 where it is evaluated by a reviewer and then included as `preliminary',
 but is not linked to the online documentation nor the command line interface.
-Inclusion of a preliminary model triggers a QC issue, and a second developer,
+Inclusion of a preliminary model 
+% ALG: automatically?
+triggers a QC issue, and a second developer,
 Developer B, then independently reimplements the model from the relevant
 primary sources and adds an automatic unit test for equality between the
-QC implementation and the preliminary production model. If the two
-implementations are equal the original model is included in \stdpopsim.
+QC implementation and the preliminary production model. 
+% ALG: How is equal defined?
+If the two implementations are equal the original model is included in \stdpopsim.
+% ALG: what does included in stdpopsim mean? made available in the next release and appears in catalog? 
```

Comment:
it means that it gets included in the documentation and appears in the catalog

Reply:
leaving as-is

Reply:
may want to add that into the text, but also maybe okay to leave as is.